BuildAddOn.py

```diff
@@ -271,46 +276,61 @@ def CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, pla
 
     elif platformName == 'MAC':
         subprocess.call ([
-            'cp', '-r',
+            'cp', '-R',
             sourceFolder / f'{addOnName}.bundle',
             packageFolder / f'{fileName}.bundle'
         ])
 
 
+def GetDevKitBuildNum (devKitData, version, platformName):
+    url = devKitData[platformName][version]
+    buildNum = url.split('/')[-2]
+    return buildNum
+
+
 # Zip packages
-def PackageAddOns (args, addOnName, platformName, acVersionList, languageList, buildFolder, packageRootFolder):
+def PackageAddOns (args, devKitData, addOnName, platformName, acVersionList, languageList, buildFolder, packageRootFolder):
     Check7ZInstallation ()
 
     for version in acVersionList:
-        if args.release:
+        if args.allLocalizedVersions:
             for languageCode in languageList:
                 CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, 'RelWithDebInfo', languageCode, True)
         else:
             CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, 'Debug')
             CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, 'RelWithDebInfo')
         
-        buildType = 'Release' if args.release else 'Daily'
-        subprocess.call ([
-            '7z', 'a',
-            str (packageRootFolder.parent / f'{addOnName}-{version}_{buildType}_{platformName}.zip'),
-            str (packageRootFolder / version / '*')
-        ])
+        if args.allLocalizedVersions:
+            buildType = 'Release'
+            subprocess.call ([
+                '7z', 'a',
+                str (packageRootFolder.parent / f'{addOnName}-{version}_{buildType}_{platformName}.zip'),
+                str (packageRootFolder / version / '*')
+            ])
+        else:
+            buildType = 'Daily'

```

Comment:
A couple of things are mixed here:
- The target localizations (`allLocalizedVersions`).
- The build frequency (`Daily`).
- `Release`, I'm not sure what does this mean.

Can't we just skip the `Release` and `Daily` addition to the name to lower the level of confusion?

Reply:
Daily contains _Debug_ and _RelWithDebInfo_ configuration builds only for the _defaultLanguage_, **Release**, however contains only _RelWithDebInfo_ configuration for _allLocalizedVersions_

Reply:
I understand, but still these are different things. I can build Debug and RelWithDebInfo monthly. I can build Debug version for all languages. Connecting these in the naming can cause confusion.

Reply:
Fixed